source/sass/cms.scss

```diff
@@ -1,6 +1,26 @@
 // Resets / sensible defaults for CMS-managed content
 
 .cms {
+  h1,
```

Comment:
These only target elements inside `.cms` tags, but the markup replacement seems like it could catch elements outside that. Is that right? What happens then?

Reply:
not right: this only affects streamfield RichtText blocks, and streamfield content always ends up in a `<div class="cms">` given the templates we've written.

Reply:
I guess it's that latter clause that concerns me slightly. But not enough to block this?

Reply:
Not sure I understand the concern: we wrote templates in a way that streamfield blocks are _always_ in a `.cms` class, and this CSS has been written to _only_ apply to content in a `.cms` class... so how would this CSS end up applying to elements outside of our cms content?

Reply:
Right, we _should_ be doing that but that's not enforced. But again, outside the scope of this issue.

Reply:
Outside the scope, but I still don't understand what you're asking. Can this affect content on our pages outside of CMS streamfields? No.

Might it in the future? Extremely unlikely, we'd have to abandon all common sense about css classes.